Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -137,6 +137,17 @@
       - NukeOpsUplink
 
+- type: listing
+  id: UplinkDualetta
+  name: uplink-dualetta-name
+  description: uplink-dualetta-desc
+  icon: { sprite: /Textures/_Goobstation/Objects/Weapons/Guns/Pistols/Dualetta.rsi, state: icon }
+  productEntity: BriefcaseSyndieDualettaBundleFilled
+  cost:
+    Telecrystal: 50
+  categories:
+  - UplinkWeaponry
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)